lib/workers/repository/init/merge.ts

```diff
@@ -220,11 +220,13 @@ export async function mergeRenovateConfig(
     logger.debug('Found npmrc in decrypted config - setting');
     npmApi.setNpmrc(decryptedConfig.npmrc);
   }
-  // Decrypt after resolving in case the preset contains npm authentication instead
-  let resolvedConfig = await decryptConfig(
```

Comment:
I think you have removed this decrypted wrapper, which is a functional and probably breaking change

Reply:
i didnt remove it,  i put it at line 229, just because of readability, 
if that's a problem ill roll it back

Reply:
I totally missed line 229 sorry. But I think best to reuse `resolvedConfig` for both line 223 and 229, because it's a `let` variable anyway

Reply:
okey, its done
i just dont like nested awaits for readability reasons